go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -70,10 +70,20 @@ type EmergencyReparentOptions struct {
 }
 
 // counters for Emergency Reparent Shard
+// TODO(timvaillancourt): move this to stats.NewCountersWithMultiLabels and drop ers_* stats in v19+.
 var (
-	ersCounter        = stats.NewGauge("ers_counter", "Number of times Emergency Reparent Shard has been run")
-	ersSuccessCounter = stats.NewGauge("ers_success_counter", "Number of times Emergency Reparent Shard has succeeded")
-	ersFailureCounter = stats.NewGauge("ers_failure_counter", "Number of times Emergency Reparent Shard has failed")
+	ersCounter = stats.NewCountersWithMultiNamesAndLabels([]string{"ers_counter", "emergency_shard_reparents"},
+		"Number of times Emergency Reparent Shard has been run",
```

Comment:
For backward compatibility, we shouldn't change the type of `ers_counter` from just a gauge to a counter that has labels associated. 
If a user has some kind of parsing setup on this metric, we don't want to break their flow. So I think we should leave all the old metrics just the way they are and introduce the new ones.

Reply:
@GuptaManan100 makes sense 👍. Updated to preserve the old stats as is